ui/app/src/components/shared/Header.vue

```diff
@@ -1,37 +1,37 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+import Menu from "./Menu.vue";
+import RowanPrice from "./RowanPrice.vue";
+
+export default defineComponent({
+  components: {
+    Menu,
+    RowanPrice
+  },
+});
+</script>
+
 <template>
   <header class="header">
     <div class="secondary-header-container">Welcome to our BetaNet! Please be aware that while this has passed several public security audits, it is still in Beta. We encourage you to first understand how it works before transacting on it and exercise caution at all times. <a target="_blank" href="https://docs.sifchain.finance/resources/betanet-launch">Here's a list of security measures taken</a>, but please note that all transactions conducted are at your own risk.</div>
     <div class="primary-header-container">
       <div>
         <Menu />
-        <!-- <SifSwitch label="Typeface Cursive" v-model="typefaceCursive" /> -->
       </div>
-      <div>
-        <slot name="right"></slot>
+      <div class="third-container">
+        <Suspense>
```

Comment:
What's `Suspense` ?

Reply:
@59023g It makes possible to use `async start()`, and does not show component until start is done loading.